test/inductor/test_torchinductor.py

```diff
@@ -12544,6 +12544,29 @@ def fn(x):
             self.assertTrue("in_out_ptr" not in code)
             self.assertEqual(fn_opt(*inps), fn(*inps))
 
+        @config.patch(inplace_buffers=True)
+        def test_layer_norm_inplaces_after_matmul(self):
```

Comment:
nit - can make test work for both devices by passing in self.device and using above